README.adoc

```diff
@@ -183,12 +182,12 @@ If you don't have an existing presentation, you can create a sample presentation
 
 To convert the sample presentation into slides, open a terminal and type:
 
- $ $(npm bin)/asciidoctor -r @asciidoctor/reveal.js -b revealjs presentation.adoc
+ $ npx asciidoctor-revealjs presentation.adoc
 
 On windows, open PowerShell and type:
 
- $ .\node_modules\.bin\asciidoctor.cmd -r @asciidoctor/reveal.js -b revealjs presentation.adoc
- 
+ $ .\node_modules\.bin\asciidoctor-revealjs.cmd presentation.adoc
```

Comment:
Will this be good for this project or did you do something special for asciidoctor.js?

Reply:
I'm not sure I quite understand your question... :thinking:

Reply:
The `.cmd` Windows file and the `\.bin\` directory are generated by npm? You didn't do anything specific to get that working?

Reply:
> The .cmd Windows file and the \.bin\ directory are generated by npm?

Yes!